control/summary_table_test.go

```diff
@@ -36,6 +36,13 @@ func docFixture(control string) *xml.XmlDocument {
 	return doc
 }
 
+func getTable(control string) xml.Node {
+	doc := docFixture(control)
+	tables, err := doc.Search("//w:tbl")
```

Comment:
nice helper! i don't know why I didn't think of this before but instead of having this custom search logic i think we should use `ssp.Document`'s `SummaryTables`